indico/migrations/versions/20240530_1743_85f58503310c_fix_reset_revisions_author.py

```diff
@@ -0,0 +1,27 @@
+"""Fix reset revisions author
+
+Revision ID: 85f58503310c
+Revises: 16c9445951f4
+Create Date: 2024-05-30 17:43:01.306657
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '85f58503310c'
+down_revision = '16c9445951f4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute('''
+        UPDATE event_editing.revisions
+        SET user_id = 0
```

Comment:
can we be sure that the "Indico System" user is always `0`? I couldn't find where in the code this is defined...

Reply:
No, we can't. It's not ID 0 on indico.cern.ch for example.

`SET user_id = (SELECT id FROM users.users where is_system)` should do the job though.

Reply:
are we sure that the system user always exists?

Reply:
Yes

Reply:
hmm, found the code where it's created in `create_all_tables`
```python
db.session.add(User(id=0, is_system=True, first_name='Indico', last_name='System'))
```
so i wonder why it's not 0 in the cern indico...

Reply:
Because there are instances which predate that function :p

On indico.cern.ch the system user was created more than a decade after the first users were being created, and unfortunately back when ZODB was a thing, the first ID was typically `0` instead of `1`, so ID 0 was already burned at that point for most objects.